doc/digraph.xml

```diff
@@ -1083,3 +1083,145 @@ gap> JohnsonDigraph(1,0);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphNumber">
+<ManSection>
+  <Oper Name="DigraphNumber" Arg="nr[, v]"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    This operation defines a bijection between the positive integers and the set
+    of all digraphs without multiple edges.  If <A>nr</A> is a positive integer,
+    then this operation returns the unique digraph which corresponds to that
+    number.  This result will be consistent across sessions and systems.  The
+    digraphs will ordered by increasing number of vertices. <P/>
+
+    If the optional argument <A>v</A> is specified and is a non-negative
+    integer, then this operation will use a different mapping which only assigns
+    numbers to digraphs with <A>v</A> vertices (with no multiple edges).  If
+    <A>nr</A> is higher than <C>2 ^ (<A>v</A> ^ 2)</C> then <K>fail</K> will be
+    returned. <P/>
+
+    See also <Ref Oper="NumberDigraph" Label="for a digraph"/>. <P/>
+
```

Comment:
It is important to mention that this **does** distinguish between isomorphic di/graphs.